src/pages/rooms/chat-input-form.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {
+  useState,
+  Dispatch,
+  SetStateAction,
+} from 'react';
+import User from '../../types/user';
+import Dropdown from './dropdown';
+
+interface Props {
+  onCreate: (input: string, callback: Dispatch<SetStateAction<string>>) => void;
+  //onEdit: (input: string, callback: Dispatch<SetStateAction<string>>) => void;
+  onCancel: (input: string) => void;
+  preSetInput: string;
+  setInput: Dispatch<SetStateAction<string>>;
+  users?: User[];
+  allowEscape: boolean
+}
+
+export default function ChatInputForm({ onCreate, onCancel, setInput, preSetInput, users, allowEscape }: Props) {
+  const [userInput, setUserInput] = useState(preSetInput);
```

Comment:
I'd recommend to rename `userInput` to just `input`, `setUserInput` to `setInput`

Reply:
Not sure about this. I have userInput for the what the user inputs and
preSetInput the text which the input form should display when there ist already something to edit
and setInput is the callback which is used  to trigger saving to the db + rerender.
The latter one would clash with your proposal. 
If you want to have the change. I would then rename the callback to setInputClbk

Reply:
The thumb up means, you want me to change names?

Reply:
it means you're right, leave it like it is. I didn't saw the prop with the name `setInput`